horace_core/sqw/file_io/@dnd_binfile_common/dnd_binfile_common.m

```diff
@@ -299,8 +298,19 @@ function check_obj_initated_properly(obj)
         % retrieve full dnd object from sqw file containing dnd or dnd and
         % sqw information
         [dnd_obj,varargout] = get_dnd(obj,varargin);
-
-
+        
+        function pix_range = get_pix_range(~)
```

Comment:
there is currently common interface for all kinds of objects as you do not know what is stored in binary file. It may be more convenient to have common interface on dnd-object pointer then catching "undefined method" on an algorithm, if you are processing a cellarray of dnd/sqw objects 

I am not entirely sure what is best as the final result will depend on new sqw object design, as In a way, dnd object would be a sqw object without pixels and without other data, so I would prefer keeping it there for the time being.  We may get rid of it later, when sqw design is fully defined.